Feat locales info options (#8985)
mdn/content: files/en-us/web/javascript/reference/global_objects/intl/locale/numberingsystems/index.md

---
title: Intl.Locale.prototype.numberingSystems
slug: Web/JavaScript/Reference/Global_Objects/Intl/Locale/numberingSystems
tags:
  - Internationalization
  - Intl
  - JavaScript
  - Property
  - Locale
  - Localization
  - Prototype
  - Reference
  - numberingSystems
browser-compat: javascript.builtins.Intl.Locale.numberingSystems
---
{{JSRef}}

> The **`Intl.Locale.prototype.numberingSystems`** property is an accessor property that returns of 1 or more unique numbering system identifiers according to the [numeral system](https://en.wikipedia.org/wiki/Numeral_system) used by the `Locale`.

Reviewer: ```suggestion
The **`Intl.Locale.prototype.numberingSystems`** property is an accessor property that returns one or more unique numbering system identifiers according to the [numeral system](https://en.wikipedia.org/wiki/Numeral_system) used by the `Locale`.
```

| sund     | Sundanese digits                                                           |
| takr     | Takri digits                                                               |
| talu     | New Tai Lue digits                                                         |
| taml     | Tamil numerals — algorithmic                                               |
| tamldec  | Modern Tamil decimal digits                                                |
| telu     | Telugu digits                                                              |
| thai     | Thai digits                                                                |
| tirh     | Tirhuta digits                                                             |
| tibt     | Tibetan digits                                                             |
| traditio | Traditional numerals — may be algorithmic                                  |
| vaii     | Vai digits                                                                 |
| wara     | Warang Citi digits                                                         |
| wcho     | Wancho digits                                                              |

## Examples

> ### Obtaining  supported `numberingSystem` values via the locale string

Reviewer: ```suggestion
### Obtaining supported `numberingSystem` values via the locale string
```

In the [Unicode locale string spec](https://www.unicode.org/reports/tr35/), the values that `numberingSystem` represents correspond to the key `nu`. `nu` is considered a locale string "extension subtag". These subtags add additional data about the locale, and are added to locale identifiers by first adding the `-u` key. To set the `numberingSystem` value via the string argument to the {{jsxref("Intl/Locale/Locale", "Locale")}} constructor, first add the `-u` extension key. Next, add the `-nu` extension key to indicate that you are adding a value for `numberingSystem`. Finally, add the `numberingSystem` value to the string.

List supported number system for a given `Locale`.

```js
let arEG = new Intl.Locale("ar-EG");

> console.log(arEG.numberingSystems); // Prints ["arab"]

Reviewer: ```suggestion
console.log(arEG.numberingSystems); // logs ["arab"]
```

```

```js
let ja = new Intl.Locale("ja");

> console.log(ja.numberingSystems); // Prints ["latn"]

Reviewer: ```suggestion
console.log(ja.numberingSystems); // logs ["latn"]
```
